scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -831,10 +836,21 @@
                 (getTaskAssigner)
                 (call task-request hostname))))))))
 
+(defn- update-host-reservations [rebalancer-reservation-atom matches]
+  (let [matched-uuids (->> matches
+                          (mapcat #(-> % :tasks))
+                          (map #(-> % .getRequest :job))
+                          (map :job/uuid)
+                          (into (hash-set)))]
```

Comment:
Can we have multiple matches for the same job?

Reply:
No